Cleanup export_model API calls

mcremon-meta opened the pull request:
Summary:
Lots of things are redundant and a few need to move to utils. Subsequent changes will split the export function and separate the run part.

Main changes:
- call `fuse_pt2` after `convert_pt2` instead of `quantize_pt2`, and avoid calling `convert_pt2` twice
- move `print_ops_info` into `export_to_cadence`
- remove the need to call `export_to_edge` in `export_model`
- move the serialization utils to `utils.py`

Differential Revision: D63795843

zonglinpeng reviewed (approved):
nice! did you try to run from the cmake side to check the CPU/HIFI flow?

mcremon-meta committed:
Cleanup export_model API calls (#5882)

Summary:

Lots of things are redundant and a few need to move to utils. Subsequent changes will split the export function and separate the run part.

Main changes:
- call `fuse_pt2` after `convert_pt2` instead of `quantize_pt2`, and avoid calling `convert_pt2` twice
- move `print_ops_info` into `export_to_cadence`
- remove the need to call `export_to_edge` in `export_model`
- move the serialization utils to `utils.py`

Reviewed By: zonglinpeng

Differential Revision: D63795843
